cost/google/instance_stackdriver_utilization/google_instance_stackdriver_utilization.pt

```diff
@@ -0,0 +1,281 @@
+name "Google Instance StackDriver Utilization"
+rs_pt_ver 20180301
+type "policy"
+short_description "Checks Google Regions and reports on StackDriver Utilization. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/google/instance_stackdriver_utilization/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
Should this be "Google Instances"? We should also include a note about the timeframe calculated.
Example: "Reports on Google instances StackDriver Utilization over the past 30 days..."